lib/src/command/outdated.dart

```diff
@@ -151,11 +152,11 @@ class OutdatedCommand extends PubCommand {
 
     rows.sort();
 
-    if (argResults['format'] == 'json') {
+    if (argResults['json']) {
       await _outputJson(rows);
     } else {
-      final useColors = argResults['format'] == 'color' ||
-          (!argResults.wasParsed('format') && stdin.hasTerminal);
+      final useColors = argResults['color'] ||
```

Comment:
Does the default in fact work? Given that `addFlag` is passed `defaultsTo: false` by default. Won't this always be `false` unless called with `--color`?

Can we detect if `--color` was passed? Or if it just assumes the default value?

Reply:
No - wasParsed('color') returns false if there was no explicit color argument.

Reply:
Can we detect if --color was passed? Or if it just assumes the default value

wasParsed('color') answers exactly that.